musicbrainzngs/musicbrainz.py

```diff
@@ -3,18 +3,20 @@
 # This file is distributed under a BSD-2-Clause type license.
 # See the COPYING file for more information.
 
+import json
 import re
 import threading
 import time
 import logging
-import socket
-import hashlib
-import locale
-import sys
-import json
 import xml.etree.ElementTree as etree
 from xml.parsers import expat
 from warnings import warn
+import requests
+from requests.auth import HTTPDigestAuth
+
+# Don't import Retry from urllib3 directly, this only works correctly the Retry
+# class form the urllib3 package requests vendors in is used.
+from requests.packages.urllib3.util.retry import Retry
```

Comment:
Ignoring the typos in the docstring above, requests recently unvendored all its external dependencies, so this code here should be double-checked (see http://docs.python-requests.org/en/latest/community/updates/#id9).